eth/gasprice/feehistory_test.go

```diff
@@ -59,7 +59,7 @@ func TestFeeHistory(t *testing.T) {
 			MaxBlockHistory:  c.maxBlock,
 		}
 		backend := newTestBackend(t, big.NewInt(16), big.NewInt(28), c.pending)
-		oracle := NewOracle(backend, config)
+		oracle := NewOracle(backend, config, nil)
```

Comment:
The nil startPrice might bring some unexpected panic. Let's sanitize the startPrice in oracle constructor